tracer/src/Datadog.Trace/Activity/Handlers/ActivityHandlerCommon.cs

```diff
@@ -104,8 +104,9 @@ public static void ActivityStarted<T>(string sourceName, T activity, OpenTelemet
                 {
                     // We ensure the activity follows the same TraceId as the span
                     // And marks the ParentId the current spanId
-
-                    if (activity.Parent is null || activity.Parent.StartTimeUtc < activeSpan.StartTime.UtcDateTime)
+                    if ((activity.Parent is null || activity.Parent.StartTimeUtc < activeSpan.StartTime.UtcDateTime)
```

Comment:
We need to do this here as well: https://github.com/DataDog/dd-trace-dotnet/blob/0f17e5756636c7ced13cdd7d003fe6dbca1039dc/tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs#L88



Also should look in other places as well just in case